File: v2/java/src/main/java/io/swagger/client/model/Meta.java
package io.swagger.client.model;

import java.util.*;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Meta  {
  
  private String sortOrder = null;
  private Boolean moreResults = null;
  private String message = null;
  private List<String> sortedBy = new ArrayList<String>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("sort_order")
  public String getSortOrder() {
    return sortOrder;
  }
  public void setSortOrder(String sortOrder) {
    this.sortOrder = sortOrder;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("more_results")
  public Boolean getMoreResults() {
    return moreResults;
  }
  public void setMoreResults(Boolean moreResults) {
    this.moreResults = moreResults;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("message")
  public String getMessage() {
    return message;
  }
  public void setMessage(String message) {
    this.message = message;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("sorted_by")
  public List<String> getSortedBy() {
    return sortedBy;
  }
  public void setSortedBy(List<String> sortedBy) {
    this.sortedBy = sortedBy;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Meta {\n");
    
    sb.append("  sortOrder: ").append(sortOrder).append("\n");
    sb.append("  moreResults: ").append(moreResults).append("\n");
    sb.append("  message: ").append(message).append("\n");
    sb.append("  sortedBy: ").append(sortedBy).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/io/swagger/Model/Meta.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Meta {
    

    
    public string SortOrder { get; set; }

    

    
    public bool? MoreResults { get; set; }

    

    
    public string Message { get; set; }

    

    
    public List<string> SortedBy { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Meta {\n");
      
      sb.Append("  SortOrder: ").Append(SortOrder).Append("\n");
      
      sb.Append("  MoreResults: ").Append(MoreResults).Append("\n");
      
      sb.Append("  Message: ").Append(Message).Append("\n");
      
      sb.Append("  SortedBy: ").Append(SortedBy).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}